camdecmps/procedures/refresh_emission_view_noxmasscems.sql

```diff
@@ -39,59 +45,130 @@ BEGIN
-		HOD.MON_PLAN_ID, 
-		HOD.MON_LOC_ID, 
-		HOD.RPT_PERIOD_ID, 
+  SELECT
+		hod.MON_PLAN_ID, 
+		hod.MON_LOC_ID, 
+		hod.RPT_PERIOD_ID, 
 		camdecmps.format_date_hour(hod.BEGIN_DATE, hod.BEGIN_HOUR, null) AS DATE_HOUR,
-		HOD.OP_TIME, 
-		HOD.HR_LOAD AS UNIT_LOAD, 
-		HOD.LOAD_UOM_CD AS LOAD_UOM, 
-		NOXC_MHV.UNADJUSTED_HRLY_VALUE AS UNADJ_NOX,
-		NOXC_MHV.APPLICABLE_BIAS_ADJ_FACTOR AS CALC_NOX_BAF,
-		NOXC_MHV.ADJUSTED_HRLY_VALUE AS RPT_ADJ_NOX,
-		NOXC_MHV.CALC_ADJUSTED_HRLY_VALUE AS ADJ_NOX_USED,
-		NOXC_MHV.MODC_CD AS NOX_MODC,
-		NOXC_MHV.PCT_AVAILABLE AS NOX_PMA,
-		FLOW_MHV.UNADJUSTED_HRLY_VALUE AS UNADJ_FLOW,
-		FLOW_MHV.APPLICABLE_BIAS_ADJ_FACTOR AS CALC_FLOW_BAF,
-		FLOW_MHV.ADJUSTED_HRLY_VALUE AS RPT_ADJ_FLOW,
-		FLOW_MHV.CALC_ADJUSTED_HRLY_VALUE AS ADJ_FLOW_USED,
-		FLOW_MHV.MODC_CD AS FLOW_MODC,
-		FLOW_MHV.PCT_AVAILABLE AS FLOW_PMA,
-		DHV.CALC_PCT_MOISTURE AS PCT_H2O_USED, 
+		hod.OP_TIME, 
+		hod.HR_LOAD AS UNIT_LOAD, 
+		hod.LOAD_UOM_CD AS LOAD_UOM, 
+		mhv.NOXC_UNADJUSTED_HRLY_VALUE AS UNADJ_NOX,
+		mhv.NOXC_APPLICABLE_BIAS_ADJ_FACTOR AS CALC_NOX_BAF,
+		mhv.NOXC_ADJUSTED_HRLY_VALUE AS RPT_ADJ_NOX,
+		mhv.NOXC_CALC_ADJUSTED_HRLY_VALUE AS ADJ_NOX_USED,
+		mhv.NOXC_MODC_CD AS NOX_MODC,
+		mhv.NOXC_PCT_AVAILABLE AS NOX_PMA,
+		mhv.FLOW_UNADJUSTED_HRLY_VALUE AS UNADJ_FLOW,
+		mhv.FLOW_APPLICABLE_BIAS_ADJ_FACTOR AS CALC_FLOW_BAF,
+		mhv.FLOW_ADJUSTED_HRLY_VALUE AS RPT_ADJ_FLOW,
+		mhv.FLOW_CALC_ADJUSTED_HRLY_VALUE AS ADJ_FLOW_USED,
+		mhv.FLOW_MODC_CD AS FLOW_MODC,
+		mhv.FLOW_PCT_AVAILABLE AS FLOW_PMA,
+		dhv.CALC_PCT_MOISTURE AS PCT_H2O_USED, 
 		CASE 
-			WHEN (DHV.CALC_PCT_MOISTURE IS NOT NULL) THEN 
+			WHEN (dhv.CALC_PCT_MOISTURE IS NOT NULL) THEN 
 				CASE 
-					WHEN (H2O_MHV.MODC_CD IS NOT NULL) THEN H2O_MHV.MODC_CD 
-					WHEN (H2O_DHV.MODC_CD IS NOT NULL) THEN H2O_DHV.MODC_CD 
+					WHEN (mhv.H2O_MODC_CD IS NOT NULL) THEN mhv.H2O_MODC_CD 
+					WHEN (dhv.H2O_MODC_CD IS NOT NULL) THEN dhv.H2O_MODC_CD 
 					ELSE 'DF'
 				END 
 			ELSE NULL
 		END AS SOURCE_H2O_VALUE, 
-		MF.EQUATION_CD AS NOX_MASS_FORMULA_CD,
-		DHV.ADJUSTED_HRLY_VALUE AS RPT_NOX_MASS,
-		DHV.CALC_ADJUSTED_HRLY_VALUE AS CALC_NOX_MASS,
-		HOD.ERROR_CODES
-	FROM temp_hourly_test_errors AS HOD 
-	INNER JOIN camdecmps.DERIVED_HRLY_VALUE  DHV 
-		ON DHV.HOUR_ID = HOD.HOUR_ID AND DHV.PARAMETER_CD = 'NOX'
-	INNER JOIN camdecmps.MONITOR_HRLY_VALUE  FLOW_MHV 
-		ON DHV.HOUR_ID = FLOW_MHV.HOUR_ID AND FLOW_MHV.PARAMETER_CD = 'FLOW'
-	INNER JOIN camdecmps.MONITOR_FORMULA  MF 
-		ON DHV.MON_FORM_ID = MF.MON_FORM_ID AND MF.EQUATION_CD LIKE 'F-26%'		
-	LEFT OUTER JOIN camdecmps.MONITOR_HRLY_VALUE  NOXC_MHV 
-		ON DHV.HOUR_ID = NOXC_MHV.HOUR_ID AND NOXC_MHV.PARAMETER_CD = 'NOXC'
-	LEFT OUTER JOIN camdecmps.MONITOR_HRLY_VALUE  H2O_MHV 
-		ON DHV.HOUR_ID = H2O_MHV.HOUR_ID AND H2O_MHV.PARAMETER_CD = 'H2O' 
-	LEFT OUTER JOIN camdecmps.DERIVED_HRLY_VALUE  H2O_DHV 
-		ON DHV.HOUR_ID = H2O_DHV.HOUR_ID AND H2O_DHV.PARAMETER_CD = 'H2O' 
+		mf.EQUATION_CD AS NOX_MASS_FORMULA_CD,
```

Comment:
The `mf` alias is undefined as you have `mf_HI` alias below which is using the `HI` parameter vs the `NOX` parameter
```
INNER JOIN camdecmps.MONITOR_FORMULA  mf_HI 
		ON mf_HI.MON_FORM_ID = dhv.MON_FORM_ID AND mf_HI.EQUATION_CD LIKE 'F-26%'
```